spec/support/common_context.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.shared_context 'with local server', :with_local_miq_server do
```

Comment:
Minor:  But could we possibly add some documentation around this?

I know it seems stupid, but it relates to my other comment about using instances variables.  Say someone "did" want to use a `@zone` variable, and you went along with the changes I suggested.  To make modifications to the context, you can do a:

```ruby
include_context :with_local_miq_server do |var_1, var_2|
  # ...
end
```

(pretty sure... I forget the DSL for it...)

Then you can even pass variables in at runtime as part of the block, allowing this to be somewhat configurable.  But without making that obvious in the docs, it might not be apparent to those who are less familiar with the whole feature set of `rspec`.

Reply:
Also:  Does this make sense to live in `EvmSpecHelper`?  Just would make it easier to find the code that this context eventually calls, and there is no reason it has to live in `support/contexts/` as far as `rspec` is concerned.

That said, if you disagree, I would argue maybe calling this `spec/support/context/with_local_miq_server.rb` instead, but that is my two cents.